packages/mobile/lib/auxiliary/drawer/stores/drawers.stores.ts

```diff
@@ -0,0 +1,4 @@
+import { writable } from 'svelte/store'
+import { IDrawer } from '../interfaces'
+
+export const drawersStore = writable<IDrawer[]>([])
```

Comment:
I think `drawers` or `drawerStates` might be a better name.

Reply:
I thought we started adding `Store` to every store to showcase is a store